cluster/manifests/job-runner/job-runner-crd.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: jobrunners.curibio.dev
+  finalizers:
+spec:
+  scope: Namespaced
+  group: curibio.dev
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                job_queue:
+                  type: string
+                ecr_repo:
+                  type: string
+                max_jobs_per_worker:
+                  type: integer
+                seconds_to_poll_db:
+                  type: integer
+
+  names:
+    kind: JobRunner
+    plural: jobrunners
+    singular: jobrunner
+    shortNames:
+      - sk
```

Comment:
I'd probably change this, I called it sk/sks because my original name for the demo code I wrote was shitkicker